### PR TITLE
Updated content in home page

### PR DESCRIPTION
**Pull request by Parvej2001**
## Fixes Issue

This PR fixes the following issues:
Updated content in home page
closes #issue-number

## Changes proposed

Here comes all the changes proposed through this PR

## Check List (Check all the boxes which are applicable) 

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Screenshots

Add all the screenshots which support your changes

### CODE REVIEW

**Review by SuperAayush — CHANGES_REQUESTED** [review]
Here are some of my suggestions!!

File: src/components/home/Content/index.jsx [hunk 1]
```diff
@@ -24,17 +24,17 @@ const Content = () => {
             btnText='Listen podcast'
           >
             <ListItem as='p'>
-              The Open Source Cafe is an initiative to highlight the stories,
-              journeys, tips & tricks of some of the amazing community leaders
-              and members.
+              The Open Source Cafe podcast is dedicated to showcasing the
+              stories, journeys, tips, and tricks of exceptional community
+              leaders and members.
```
Comment (siddhant-khisty) [inline src/components/home/Content/index.jsx:29]: The Open Source Cafe podcast is committed to presenting the narratives, experiences, insights, and techniques of outstanding leaders and members within the technical community.

File: src/components/home/Content/index.jsx [hunk 2]
```diff
@@ -24,17 +24,17 @@ const Content = () => {
...
-              Learn from the expereinces of industry expert, and get mentored
-              from folks around the globe.
+              Tune in to learn from industry experts and get mentored by
+              individuals from around the world.
             </ListItem>
             <ListItem as='p'>
-              Grow your skills to the next level by gather tricks and tips from
-              folks who have made it.
+              Elevate your skills with insights and advice from successful
+              individuals who have achieved great heights in their careers.
```
[the remark this answers is not in the document]
Comment (siddhant-khisty) [reply src/components/home/Content/index.jsx:37]: `accomplished individuals`*

File: src/components/home/Content/index.jsx [hunk 3]
```diff
@@ -44,19 +44,19 @@ const Content = () => {
             btnText='Read and learn'
           >
             <ListItem as='p'>
-              <span className='highlight'>KubeLetter</span> - We invite you to
-              check out our kubeletters, where we provide updates and insights
-              on all about cloud native.
+              <span className='highlight'>KubeLetter</span> - Check out our
+              KubeLetter blog for updates and insights on all things related to
+              cloud native technology.
```
Comment (siddhant-khisty) [inline src/components/home/Content/index.jsx:49]: Check out the Kubeletter, our monthly newsletter covering updates, insights, and sharing valuable resources around all things cloud native.

File: src/components/home/Content/index.jsx [hunk 4]
```diff
@@ -44,19 +44,19 @@ const Content = () => {
...
             </ListItem>
             <ListItem as='p'>
-              <span className='highlight'>Case Studies</span> - Our case studies
-              feature real-world use cases and the influence of various
-              technologies on enterprises.
+              <span className='highlight'>Case Studies</span> - Explore our
+              collection of case studies, featuring real-world examples of how
+              different technologies have impacted enterprises.
```
Comment (siddhant-khisty) [inline src/components/home/Content/index.jsx:54]: Explore our collection of case studies, featuring real-world examples of how different technologies have impacted developers, making their lives easier.

File: src/components/home/Mentorship/index.jsx [hunk 5]
```diff
@@ -19,16 +19,17 @@ const Mentorship = ({ background = 'static' }) => {
...
       <ListItem>
         <span className='highlight higlight-white'>
-          A comprehensive answer to all of your mentoring needs
+          Comprehensive Mentorship:
         </span>
-        , including educational materials, communities to join, opportunities,
-        and much more.
+        We provide a one-stop solution for all your mentoring needs, including
+        educational resources, communities to join, and opportunities to enhance
+        your skills.
```
[the remark this answers is not in the document]
Comment (siddhant-khisty) [reply src/components/home/Mentorship/index.jsx:26]:
```
including educational resources, domain roadmaps, communities to join, and many great opportunities to improve your abilities.
```

File: src/components/home/Mentorship/index.jsx [hunk 6]
```diff
@@ -19,16 +19,17 @@ const Mentorship = ({ background = 'static' }) => {
...
       </ListItem>
       <ListItem>
-        <span className='highlight higlight-white'>
-          Roadmaps to help you stay on track
-        </span>
-        , including those for open-source, full-stack, devrel, and more.
+        <span className='highlight higlight-white'>Customized Roadmaps:</span>,
+        We offer personalized roadmaps to help you stay on track, whether you
```
Comment (siddhant-khisty) [inline src/components/home/Mentorship/index.jsx:30]:
```suggestion
        We offer domain-specific roadmaps to help you stay on track, whether you
```

File: src/components/home/about/index.jsx [hunk 7]
```diff
@@ -29,36 +29,20 @@ const About = () => {
...
-          We provide{' '}
-          <span className='highlight highlight--white'>
-            hands-on training, mentorship for FREE
-          </span>{' '}
-          and have an{' '}
-          <span className='highlight highlight--white'>
-            inclusive community.
-          </span>
+          We offer free hands-on training and mentorship programs, as well as an
+          inclusive community.{' '}
```
[the remark this answers is not in the document]
Comment (siddhant-khisty) [reply src/components/home/about/index.jsx:37]: We provide free practical training, mentorship programs, many learning opportunities, and a welcoming community with over 60,000 members.

File: src/components/home/about/index.jsx [hunk 8]
```diff
@@ -29,36 +29,20 @@ const About = () => {
...
+          Our services include expert guidance on career development, Open
+          Source, and global internships and job opportunities.{' '}
         </ListItem>
         <ListItem>
-          We are also having various{' '}
-          <span className='highlight highlight--white'>
-            events weekly or monthly.{' '}
-          </span>
-          Do participate in those events to make your skills enhance.
+          We also organize regular weekly and monthly events to help individuals
```
Comment (siddhant-khisty) [inline src/components/home/about/index.jsx:44]:
```suggestion
          We also organize regular events to help individuals
```

File: src/components/home/courses/index.jsx [hunk 9]
```diff
@@ -22,15 +22,19 @@ const Courses = ({ background = 'static' }) => {
         background={background}
       >
         <ListItem>
-          All courses are FREE while never compromising on quality.
+          High-Quality Courses for Free: Our courses never compromise on
+          quality, yet they are entirely free for all learners.
```
[the remark this answers is not in the document]
Comment (siddhant-khisty) [reply src/components/home/courses/index.jsx:26]: `completely free for everyone, without sacrificing on quality`*

File: src/components/home/events/index.jsx [hunk 10]
```diff
@@ -31,24 +31,16 @@ const Events = () => {
         direction='right'
       >
         <ListItem>
-          <span className='highlight highlight--white'>
-            All of our events are completely free and open for everyone.
-          </span>{' '}
-          Fun-filled events with no compromise on quality.
+          Our events are free and open to everyone. We believe in organizing
+          fun-filled events without compromising on quality.
```
Comment (siddhant-khisty) [inline src/components/home/events/index.jsx:35]:
```suggestion
          fun-filled events that do not have any entry-barriers without compromising on quality.
```

File: src/components/home/events/index.jsx [hunk 11]
```diff
@@ -31,24 +31,16 @@ const Events = () => {
...
+          Our events help students develop various co-curricular skills and
+          expose them to the latest trends in technical and professional fields.
         </ListItem>
         <ListItem>
-          Explore the plethora of events & have the opportunity to{' '}
-          <span className='highlight highlight--white'>
-            grab amazing prizes & goodies!
-          </span>
+          Don't miss the chance to explore a range of events and win amazing
+          prizes and goodies!
```
Comment (siddhant-khisty) [inline src/components/home/events/index.jsx:43]:
```suggestion
          Don't miss the chance to explore a range of events, improve your skills and win amazing
          prizes and goodies!
```

**Comment by tejash023** [comment]
hi @SuperAayush Can I work on this? or is this completed?

**Pull request closed without merging**